### ISSUE TITLE
strip ?request=getcapabilities from request before add request=getmap

### ISSUE THREAD

**Issue by pvgenuchten** [meggsimum/wegue#287]
not sure if this is an openlayers or wegue issue

if you add a wms layer by url example.com/ows?service=wms&request=getcapabilities

the viewer adds; `request=getmap` to the querystring, without removing `request=getcapabilities`

just to verify, the getmap request is send to the submitted getcapabilites url or the onlineresource as advertised in capabilities?

**Comment by JakobMiksch** [comment]
I can reproduce it with

  ```
 {
      "type": "WMS",
      "lid": "ahocevar-wms",
      "format": "image/png",
      "layers": "topp:states",
      "url": "https://ahocevar.com/geoserver/wms?SERVICE=WMS&REQUEST=GetCapabilities",
      "XXXurl": "https://ahocevar.com/geoserver/wms",
      "transparent": true,
      "singleTile": false,
      "projection": "EPSG:3857",
      "attribution": "",
      "isBaseLayer": false,
      "visible": true,
      "displayInLayerList": true
    },
```
As far I can see from the code (https://github.com/wegue-oss/wegue/blob/7f53a55545015d37c5b4fc73c8f9dba334240b7d/src/factory/Layer.js#L107) the url is taken as is and no parsing of the capabilities is done.

Does it work for you to just skip the query params of your provided url?

**Comment by pvgenuchten** [comment]
Yes it works without the query, a typical use case is to find a record in a catalogue, take the wms url from the record as-is, this url often has the ?request=getcapabilities postfix, splitting at ? Is not a good idea, because some wms urls include a required parameter, like mapserver; map=foo.map
